mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
```

Comment:
The `findOneAndDelete` method returns a document, not a boolean value. It's possible that `deletionResult` could be an empty object `{}` if the order is deleted successfully, but the document is not found in the database. 

It would be better to check if `deletionResult.ok` is 1 and `deletionResult.deletedCount` is 1 to ensure the order was deleted successfully.